Docs: Update documentation on PDF/Email generation with logo (#114005)
grafana/grafana: docs/sources/visualizations/dashboards/create-reports/report-settings/index.md

- **Company logo** - Company logo displayed in the report PDF.
  Configure it by specifying a URL or uploading a file.
  The maximum file size is 16 MB.

>   If not set, defaults to the Grafana logo. If the URL specified is not correct, the logo will appear as broken link logo.

Reviewer: ```suggestion
  If not set, defaults to the Grafana logo. If the URL specified isn't correct, the logo appears as broken link logo.
```

Reviewer: maybe "isn't correct" is a bit vague - how about something like "isn't a valid image"

<!-- vale Grafana.WordList = YES -->

> - **Company logo** - Company logo displayed in the report email. Configure it by specifying a URL or uploading a file. The maximum file size is 16 MB. If not set, defaults to the Grafana logo. If the URL specified is not correct, the logo will appear as a broken link logo.

Reviewer: I feel the same here as my previous comment. It's not a huge deal but maybe we can be more clear
